### DISCUSSION TITLE
Request for Assistance Due to Lost Access with 2FA

### DISCUSSION THREAD

**Discussion by eugenioand19** [community/community#138132] [New to GitHub]
Dear GitHub Support Team,

I am reaching out for assistance regarding an issue with my account. My mobile phone, which I used for two-factor authentication (2FA), was stolen. Unfortunately, I no longer have access to my recovery codes or any alternative method to log into my account.

The affected account is linked to my education email, but I am currently unable to access it in any way. Due to the total loss of my authentication devices and methods, I am locked out of my account.

I would greatly appreciate your help in restoring my access. If you require any further information to verify my identity and ownership of the account, I am willing to provide it.

Thank you in advance for your time and assistance.

**Comment by VadymMakohon** [comment]
In this situation, since you've lost access to your 2FA method and recovery codes, the best course of action is to contact GitHub Support directly to request account recovery. Follow these steps:

Steps to recover your account:
Submit a Support Ticket:

Go to the [GitHub Support page](https://support.github.com/contact) and select the "Account Recovery" option.
Clearly explain your situation: mention that your phone was stolen, you’ve lost access to 2FA, and you no longer have your recovery codes.
Provide Proof of Ownership:

In the support request, provide as much information as possible to verify your identity and ownership of the account. This can include:
The email address linked to your account (your education email).
Details about the repositories you own or collaborate on.
The last date you successfully logged in or any other identifying information.
Wait for Support Response:

GitHub will guide you through their verification process. This may take some time depending on the volume of requests.
Unfortunately, without access to 2FA or recovery codes, GitHub is the only way to regain access to your account. Be patient as they handle the recovery process.
